deps/hiredis/COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2009-2011, Salvatore Sanfilippo <antirez at gmail dot com>
+Copyright (c) 2009-2011, Redis Ltd.
```

Comment:
Redis Ltd founded in 2011 (https://en.wikipedia.org/wiki/Redis_(company)), could you please update this properly? There are some similar updates in other files, Thanks